tools/src/tester/start.ts

```diff
@@ -0,0 +1,21 @@
+import OpenApiMerger from '../merger/OpenApiMerger'
+import { LogLevel } from '../Logger'
+import TestsRunner from './TestsRunner'
+import { Command, Option } from '@commander-js/extra-typings'
+
+const command = new Command()
+  .description('Run test stories against the OpenSearch spec.')
+  .addOption(new Option('--spec_path <path>', 'path to the root folder of the multi-file spec').default('./spec'))
```

Comment:
`--ignore-passed` meaning ignore printing PASSED chapters . You can turn it on so you can see all chapters being printed out when one of the chapters failed instead of just the failed one.

Reply:
Went Redid the hide/show logic and apply 1 single rule across the board: If a component is either FAILED or ERROR, all of its sub-components will be displayed. The `--ignore` option has been removed.

Reply:
![image](https://github.com/opensearch-project/opensearch-api-specification/assets/644650/22d131a9-7fb3-4733-8282-6ce4de947ca6)